scripts/enum/mod.lua

```diff
@@ -410,6 +410,10 @@ xi.mod =
     DARK_MAGIC_DURATION             = 1072, -- Increases Dark Magic spell durations by percentage (e.g. mod value 10 = +10% duration)
     ENHANCES_DARK_SEAL              = 1073, -- Enhances "Dark Seal" effect (Increases Dark Magic spell durations by 10% per Dark Seal merit while Dark Seal active)
 
+    -- Beastmaster
+    TANDEM_STRIKE                   = 271,  -- Grants a bonus to your and your pet's accuracy and magic accuracy when you and your pet are attacking the same target.
```

Comment:
Considering this is a modifier and not the trait itself, can we name this `TANDEM_STRIKE_BONUS`?

Reply:
`TANDEM_STRIKE_ACURACY` fits better I think, bonus is usually a loaded word for things adding onto when we're naming mods

Reply:
except ya know, spell better than teo..

Reply:
likewise `TANDEM_BLOW_TP_REDUCTION` even if a bit wide.

Reply:
Well, considering its a bonus to both ACC and MACC, i thought a more generic name would fit better. Not opposd to it though

Reply:
ok then, lets go with `TANDEM_STRIKE_ACCURACY` and `TANDEM_BLOW_TP_REDUCTION`

Reply:
good point about it not being the real mod, i went with `_POWER` to indicate we're just defining how strong it is

Reply:
`_POWER` sounds alright but might get changed somewhere down the road depending on future bs SE adds

Reply:
(if `_ACCMACC` or even `_ACC` happened I'd be gettin muh paddle)